docs/rules/README.md

```diff
@@ -8,266 +8,265 @@ Some rules are fixable using the `--fix` command line flag. Those rules are mark
-* [operator-assignment](operator-assignment.md): require assignment operator shorthand where possible or prohibit it entirely
-* [operator-linebreak](operator-linebreak.md): enforce operators to be placed before or after line breaks
-* [padded-blocks](padded-blocks.md): enforce padding within blocks
+* [one-var](one-var.md): enforce variables to be declared at the same time or separately per function
+* [one-var-declaration-per-line](one-var-declaration-per-line.md): require or disallow newlines around variable declarations
+* [operator-assignment](operator-assignment.md): require or disallow assignment operator shorthand where possible
+* [operator-linebreak](operator-linebreak.md): require operators to be placed before or after line breaks
+* [padded-blocks](padded-blocks.md): require or disallow padding within blocks
 * [quote-props](quote-props.md): require quotes around object literal property names
-* [quotes](quotes.md): specify whether backticks, double or single quotes should be used (fixable)
-* [require-jsdoc](require-jsdoc.md): require JSDoc comment
-* [semi](semi.md): require or disallow use of semicolons instead of ASI (fixable)
-* [semi-spacing](semi-spacing.md): enforce spacing before and after semicolons (fixable)
-* [sort-imports](sort-imports.md): enforce sorting import declarations within module
-* [sort-vars](sort-vars.md): enforce sorting variables within the same declaration block
-* [space-before-blocks](space-before-blocks.md): require or disallow a space before blocks (fixable)
-* [space-before-function-paren](space-before-function-paren.md): require or disallow a space before function opening parenthesis (fixable)
-* [space-in-parens](space-in-parens.md): require or disallow spaces inside parentheses (fixable)
-* [space-infix-ops](space-infix-ops.md): require spaces around operators (fixable)
-* [space-unary-ops](space-unary-ops.md): require or disallow spaces before/after unary operators (fixable)
-* [spaced-comment](spaced-comment.md): require or disallow a space immediately following the `//` or `/*` in a comment (fixable)
-* [wrap-regex](wrap-regex.md): require regex literals to be wrapped in parentheses
+* [quotes](quotes.md): enforce the consistent use of either backticks, double, or single quotes (fixable)
+* [require-jsdoc](require-jsdoc.md): require JSDoc comments
+* [semi](semi.md): require or disallow semicolons instead of ASI (fixable)
+* [semi-spacing](semi-spacing.md): require or disallow spacing before and after semicolons (fixable)
+* [sort-imports](sort-imports.md): enforce sorted import declarations within module
+* [sort-vars](sort-vars.md): enforce sorted variables within the same declaration block
+* [space-before-blocks](space-before-blocks.md): require or disallow spacing before blocks (fixable)
+* [space-before-function-paren](space-before-function-paren.md): require or disallow spacing before opening function parentheses (fixable)
```

Comment:
enforce consistent spacing before opening parenthesis in function definitions

Trying to distinguish from no-spaced-func which deals with function invocations. Please improve!